resources/sdk/webmessagingjava/templates/WebMessagingClient.mustache

```diff
@@ -41,6 +41,27 @@ public class WebMessagingClient {
     private final ArrayList<SessionListener> sessionListeners = new ArrayList<>();
     private ApiClient apiClient;
 
+    private AllowedMedia allowedMediaInbound;
+
+    public AllowedMedia getAllowedMediaInbound() {
+        return allowedMediaInbound;
+    }
+
+    public void setAllowedMediaInbound(AllowedMedia allowedMediaInbound) {
+        this.allowedMediaInbound = allowedMediaInbound;
+    }
+
+    /**
+     * check if a provided mime type is among the allowed one in configuration for inbound content.
+     * @param mimeType
+     * @return true if the mime type is part of the allowed media, false otherwise
+     * @see #getSupportedContentConfiguration(String, String, String)
+     * @see #allowedMediaInbound
+     */
+    public boolean isMimeTypeAllowedInbound(String mimeType) {
```

Comment:
how is this method being called. and how are we making sure allowedMediaInbound is initialised before calling this.

Reply:
It's filled in when a response has been received [line 648](https://github.com/gartcimore/platform-client-sdk-common/blob/750bf5a96575d581258d8376419a944c989f85d6/resources/sdk/webmessagingjava/templates/WebMessagingClient.mustache#L648) and [line 690](https://github.com/gartcimore/platform-client-sdk-common/blob/750bf5a96575d581258d8376419a944c989f85d6/resources/sdk/webmessagingjava/templates/WebMessagingClient.mustache#L690
)